template/clients/rust/scripts/client/lint-rust.mjs

```diff
@@ -1,7 +1,28 @@
 #!/usr/bin/env zx
 import 'zx/globals';
-import { workingDirectory } from '../utils.mjs';
+import {
+  cliArguments,
+  getToolchainArgument,
+  popArgument,
+  workingDirectory,
+} from '../utils.mjs';
+
+// Configure additional arguments here, e.g.:
+// ['--arg1', '--arg2', ...cliArguments()]
+const lintArgs = cliArguments();
+
+const fix = popArgument(lintArgs, '--fix');
+const toolchain = getToolchainArgument('format');
+const manifestPath = path.join(
+  workingDirectory,
+  'clients',
+  'rust',
+  'Cargo.toml'
+);
 
 // Check the client using Clippy.
-cd(path.join(workingDirectory, 'clients', 'rust'));
-await $`cargo clippy ${process.argv.slice(3)}`;
+if (fix) {
+  await $`cargo ${toolchain} clippy --manifest-path ${manifestPath} --fix ${lintArgs}`;
+} else {
+  await $`cargo ${toolchain} clippy --manifest-path ${manifestPath} ${lintArgs}`;
+}
```

Comment:
Clippy is even less clear on which arguments are which, but it also uses a `--` delimiter. I think the separation is similar to `cargo fmt `/`rustfmt`, but `clippy` isn't its own command.

Reply:
Less of a problem with the clippy script, though since we're not including `--` in the default.